modules/installation/proc-installing-rhdh-on-openshift-with-helm-cli.adoc

```diff
@@ -0,0 +1,45 @@
+[id="proc-installing-rhdh-on-openshift-with-helm-cli_{context}"]
+= Installing {product} on {ocp-brand-name} with Helm CLI
+
+You can use the Helm CLI to install {product} on {ocp-brand-name}.
+
+.Prerequisites
+* An `oc` session, with `developer` or `admin` permissions, on an {ocp-short} cluster.
+* You have installed the link:https://helm.sh/docs/intro/install[Helm CLI].
+
+.Procedure
```

Comment:
This would add undesired fluff. The examples in the supplementary style guide do not add this. https://redhat-documentation.github.io/supplementary-style-guide/#commands-in-code-blocks